failed to make openblas  in centos7 of aarch64

axpy.c:1:0: error: unknown value ‘cortex-a72’ for -mtune
 /*********************************************************************/
 ^
copy.c:1:0: error: unknown value ‘cortex-a72’ for -mtune
 /*********************************************************************/
 ^
swap.c:1:0: error: unknown value ‘cortex-a72’ for -mtune
 /*********************************************************************/
 ^
make[1]: *** [saxpy.o] Error 1
make[1]: *** Waiting for unfinished jobs....
scal.c:1:0: error: unknown value ‘cortex-a72’ for -mtune
 /*********************************************************************/
 ^
make[1]: *** [sswap.o] Error 1
make[1]: *** [scopy.o] Error 1
make[1]: *** [sscal.o] Error 1
make[1]: Leaving directory `/root/OpenBLAS/interface'
make: *** [libs] Error 1

--- a/Makefile.arm64
+++ b/Makefile.arm64
@@ -1,4 +1,15 @@
 ifneq ($(C_COMPILER), PGI)
+
+ifneq ($(GCCVERSIONGT4), 1)
+CCOMMON_OPT += -march=armv8-a
+ifneq ($(F_COMPILER), NAG)
+FCOMMON_OPT += -march=armv8-a
+endif
+
+
+else 
+
+
 ifeq ($(CORE), ARMV8)
 CCOMMON_OPT += -march=armv8-a
 ifneq ($(F_COMPILER), NAG)
@@ -138,4 +149,7 @@ FCOMMON_OPT += -march=armv8-a -mtune=emag
 endif
 endif
 endif
+
+endif
+
 endif

